### ISSUE TITLE
[4.0] npm run build:css

### ISSUE THREAD

**Issue by rjharishabh** [joomla/joomla-cms#35934]
### Steps to reproduce the issue
Do `npm run build:css`

### Expected result
No ERROR

### Actual result
ERROR
```
> joomla@4.0.0 build:css
> node build/build.js --compile-css

Error: Undefined variable.
   ╷
20 │   @each $color, $value in $theme-colors-rgb {
   │                           ^^^^^^^^^^^^^^^^^
   ╵
  administrator\templates\atum\scss\_root.scss 20:27            @import
  administrator\templates\atum\scss\vendor\_bootstrap.scss 8:9  @import
  administrator\templates\atum\scss\template.scss 8:9           @import
  administrator\templates\atum\scss\template-rtl.scss 1:9       root stylesheet
npm ERR! code 1
npm ERR! path D:\Xampp\htdocs\joomla
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node build/build.js --compile-css

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Rishabh\AppData\Local\npm-cache\_logs\2021-10-29T14_32_57_618Z-debug.log
```

### System information (as much as possible)
Chrome on Windows10 
PHP 8.0.6
MySQLi 10.4.19-MariaDB

**Comment by brianteeman** [comment]
Unable to replicate

> npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Rishabh\AppData\Local\npm-cache\_logs\2021-10-29T14_32_57_618Z-debug.log


This log should give you more information

**Comment by khu5h1** [comment]
Yes, I'm also unable to replicate the issue. I think this issue might be system-specific.

**Comment by richard67** [comment]
@rjharishabh Just to be sure: Have you run „composer install“ before running „npm ci“?

**Comment by rjharishabh** [comment]
No ERROR in fresh Joomla install, but there are deprecation warnings

```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($col-gutter-width, 2)

More info and automated migrator: https://sass-lang.com/d/slash-div
```

**Comment by richard67** [comment]
The warnings about deprecated math division are a known issue. There’s a PR to fix that, #35648 , but there might be better ways to fix that.
